examples/Components/topology/TopologicalModifiers/RemovingTriangle2EdgeProcess.scn

```diff
@@ -27,7 +28,7 @@
     <MinProximityIntersection name="Proximity" alarmDistance="0.8" contactDistance="0.5" />
     <Node name="SquareGravity">
         <EulerImplicitSolver name="cg_odesolver" rayleighStiffness="0.1" rayleighMass="0.1" />
-        <CGLinearSolver iterations="25" name="linear solver" tolerance="1e-5" threshold="1e-5"/>
+        <CGLinearSolver iterations="25" name="linear solver" tolerance="1e-5" threshold="1e-10"/>
```

Comment:
why?

Reply:
When running using the value 1e-5 we obtain the following:
```
[WARNING] [CGLinearSolver(linear solver)] denominator threshold reached at first iteration of CG  
Check the 'threshold' data field, you might decrease it
malloc_consolidate(): invalid chunk size
```

Reply:
👍